.github/workflows/snyk.yaml

```diff
@@ -0,0 +1,36 @@
+name: Snyk test Docker image
+on: 
+  push:
+    branches: [master]
+  workflow_dispatch:
+
+jobs:
+  snyk_test:
+    name: Snyk test
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v3
+
+    - name: Load openvds image
+      uses: "./.github/actions/load_openvds_image"
+
+    - name: Build docker image
+      run: |
+        tag=snyk_server
```

Comment:
Why is our image for testing named snyk server?
It shows up in the logs and is confusing

> Testing snyk_server...

Reply:
we can change the name. I just added this tag to distinguish the image name.